src/drive/web/modules/drive/Toolbar/components/CreateNoteItem.jsx

```diff
@@ -1,16 +1,31 @@
 import React from 'react'
+import get from 'lodash/get'
+
 import { translate } from 'cozy-ui/transpiled/react/I18n'
-import toolbarContainer from 'drive/web/modules/drive/Toolbar/toolbar'
+import { withClient, models, useAppLinkWithStoreFallback } from 'cozy-client'
+import { generateUniversalLink } from 'cozy-ui/transpiled/react/AppLinker/native'
 
+import toolbarContainer from 'drive/web/modules/drive/Toolbar/toolbar'
 import styles from 'drive/styles/toolbar.styl'
-import { withClient, models, useAppLinkWithStoreFallback } from 'cozy-client'
+import useCapabilities from 'lib/hooks/useCapabilities'
 
 const CreateNoteItem = ({ client, t, displayedFolder }) => {
   const { fetchStatus, url, isInstalled } = useAppLinkWithStoreFallback(
     'notes',
     client
   )
+  const capabilities = useCapabilities(client)
```

Comment:
Friendly reminder that useCapabilities does not use a fetchPolicy meaning that we fetch the settings every time a CreateNoteItem is displayed.

Reply:
It's displayed all the time, so only one request or one more if the client change ;). 

I'm wondering if I should move this hook to cozy-client in order to not repeat that everywhere and also if one day we have a useQuery() with fetch policy we could be able to update all our app with that :)

Reply:
I think it has its place in cozy-client, yes :)